src/etc/inc/plugins.inc.d/unbound.inc

```diff
@@ -508,6 +508,9 @@ function unbound_add_domain_overrides($pvt = false)
         if ($pvt == true) {
             $domain_entries .= "private-domain: \"$domain\"\n";
             $domain_entries .= "domain-insecure: \"$domain\"\n";
+            if ((substr($domain, -14) == ".in-addr.arpa.") || (substr($domain, -13) == ".in-addr.arpa")) {
```

Comment:
cool :) as a small follow up, maybe we should only match non-empty left side with .+  ?